src/main/java/ro/societateahermes/backendservice/controller/FormController.java

```diff
@@ -0,0 +1,53 @@
+package ro.societateahermes.backendservice.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.societateahermes.backendservice.entities.dto.FormDto;
+import ro.societateahermes.backendservice.entities.form.Form;
+import ro.societateahermes.backendservice.service.serviceImplementation.FormService;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/form")
+public class FormController {
+    private final FormService formService;
+
+    public FormController(FormService formService) {
+        this.formService = formService;
+    }
+
+    @GetMapping
+    public List<FormDto> getAll() {
+        List<Form> forms = formService.getAll();
+        return forms.stream()
+                .map(this::convertToDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public void save(@RequestBody FormDto formDto) {
+        formService.save(this.convertToEntity(formDto));
+    }
+
+    @DeleteMapping("/{formId}")
+    public void delete(@PathVariable("formId") Long formId) {
+        formService.delete(formId);
+    }
+
+    private FormDto convertToDto(Form form) {
```

Comment:
Metodele de conversie nu au ce sa caute in controller. El ar trebui sa aibe o singura responsabilitate, anume sa intercepteze requesturi si sa raspunda la ele. In momentul in care tu pui metodele astea in el, capata o a doua responsabilitate, ceea ce o sa-ti faca viata mai grea mai tarziu in ciclul de viata al aplicatiei. Metodele de mapare ar trebui sa fie implementate in interiorul unei clase care sa se cheme FormMapper sau ceva de genul pentru ca asa vei putea refolosi codul, nu vei pierde maparea o data cu controllerul daca il schimbi